src/molecules/card-header/index.jsx

```diff
@@ -1,9 +1,22 @@
 import CardMedia from "@mui/material/CardMedia";
 import * as React from "react";
+import useStore from "../../ions/store/store";
+import VocabularyNoHolder from "../../atoms/vocabulary-no-holder";
 
 const CardHeader = ({ ...props }) => {
+	const currentCard = useStore(state => state.currentCard);
 	return (
-		<CardMedia component="img" height="140" image="/images/flashcard.png" alt={props.lesson} />
+		<>
+			<CardMedia
+				sx={{ marginY: 0 }}
+				component="img"
+				height="auto"
+				image="/images/flashcard.png"
+				alt={props.lesson}
+				// number={currentCard.vocabularyNo}
```

Comment:
better to not keep unused code

Reply:
Ok, thanks. will delete on next round. :)